spec/acceptance/new_class_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper_acceptance'
+
+describe 'Creating a new class' do
+  before(:all) do
+    shell_ex("#{path_to_pdk} new module foo --skip-interview")
+  end
+
+  after(:all) do
+    shell_ex('rm -rf foo')
+  end
+
+  context 'when creating the main class' do
+    describe command("cd foo && BUNDLE_GEMFILE=../Gemfile #{path_to_pdk} new class foo") do
```

Comment:
I assume you meant @scotje ?

Yeah binstubs look like a good way out here - I'll try that. It might also help with the bundle/rvm fighting we ran into yesterday...